libraries/util/appCommon/adminKpc.py

```diff
@@ -1244,4 +1244,157 @@ def searchCustomer(page, strEmail):
         uCommon.setElem(page, pAdmin.cu.searchByTxt, strEmail)
         uCommon.waitAndClickElem(page, pAdmin.cu.searchIconBtn)
         uCommon.wait(page, 2)
-        uCommon.waitAndClickElem(page, pAdmin.cu.customerResultBtn)
\ No newline at end of file
+        uCommon.waitAndClickElem(page, pAdmin.cu.customerResultBtn)
+        
+    @uCommon.ufuncLog  
+    def verifyUploadRewardsOrCreditButton(page):
+        """ 
+        Objective: Verify that Upload Rewards or Credit button is displayed.
+        
+        param: None
+        returns: None
+        Author: abernal_20240318
+        """
+        if uCommon.verifyVisible(page, pAdmin.cu.uploadCreditRewardBtn) == True:
+            uCommon.log(1, f'Upload Rewards or Credit button is visible.') 
+        elif uCommon.verifyVisible(page, pAdmin.cu.uploadCreditRewardBtn) == False:
+            uCommon.log(2, f'Upload Rewards or Credit button is not visible.') 
+            
+    @uCommon.ufuncLog  
+    def verifyTotalCreditAndRewardFields(page):
+        """ 
+        Objective: Verify that Total Credit and Total Reward fields are displayed.
+        
+        param: None
+        returns: None
+        Author: abernal_20240318
+        """
+        if uCommon.verifyVisible(page, pAdmin.cu.totalCreditLbl) == True:
+            uCommon.log(1, f'Total Credit is visible.') 

```

Comment:
Pls add condition to 'False' state so we can have log if Total credit is not visible.